geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AccumuloFeatureIndex.scala

```diff
@@ -90,18 +92,22 @@ object AccumuloFeatureIndex extends AccumuloIndexManagerType with LazyLogging {
 
   /**
     * Maps a simple feature type to a set of default indices based on
-    * attributes and when it was created (schema version)
+    * attributes and when it was created (schema version).
+    *
+    * Note that schema version has been deprecated and this method should not be called
+    * except when transitioning from schema version to per-index versions.
```

Comment:
I just wanted to make sure no one calls this method accidentally - it's only for a particular use case (transitioning an old schema version).